notebook/static/edit/js/savewidget.js

```diff
@@ -122,7 +122,7 @@ define([
         if(filename){
             this._filename = filename;
         }
-        document.title = (dirty?'*':'')+this._filename;
+        document.title = (dirty?'*':'')+this._filename+' - Jupyter Notebook';
```

Comment:
I wonder if this is confusing on the text editor. It's a component of the Jupyter Notebook application, but the file in this tab is not *a Jupyter notebook*.

Reply:
Changed to "Jupyter Text Editor".